### ISSUE TITLE
UserVoice Idea: COURSERA VERIFIED CERTIFICATE NO LONGER FREE FOR VETS

### ISSUE THREAD

**Issue by MarinaMartin** [department-of-veterans-affairs/vets-website#995]
Some transitioning members in San Diego logged in to the Coursera blog yesterday. It said, "Sorry the voucher no longer exists." Do the VA and Coursera working on extending the free verified Coursera certificate voucher for veterans?

http://veterans.uservoice.com/forums/330333-vets-gov/suggestions/11325942-coursera-verified-certificate-no-longer-free-for-v

**Comment by koconnor719** [comment]
when i click on courser from my VA machine i get this error:

<img width="1123" alt="screen shot 2016-01-07 at 1 27 45 pm" src="https://cloud.githubusercontent.com/assets/13770771/12179459/03908078-b545-11e5-9090-bc681637515c.png">

**Comment by koconnor719** [comment]
alex, do you want me to create a new issue in VEC repo?  or is this one sufficient? 

@ayaleloehr

**Comment by ayaleloehr** [comment]
Having the Coursera ones together is sufficient. We have en email out to Coursera asking about this. Will update when we hear back.

**Comment by koconnor719** [comment]
posted response:

Thank you for your feedback. We are checking on this.

vets.gov team

**Comment by ayaleloehr** [comment]
@koconnor719 for documentation, per our call with Coursera, this error message occurs when a user has already claimed their voucher (they can claim it only once).

**Comment by koconnor719** [comment]
posted response; note: Coursera is fixing the blog issue above by using survey monkey instead of google docs (issue was an access issue specific to VA computers and Google docs:)

Thank you for your feedback. Our team did some testing on this. Users can claim a Coursera voucher only once. That error message, "sorry the voucher no longer exists" happens when a user has already claimed a voucher and tries to claim it again. Were you able to claim your voucher the first time?  

vets.gov team

**Comment by koconnor719** [comment]
coursers still working to fix the broken log (replacing google doc w/ survey monkey) this issue is limited to VA users only

**Comment by koconnor719** [comment]
escalated to Coursera

**Comment by ayaleloehr** [comment]
Just checked the UserVoice comments on this issue.  The user reported "I have been unable to get to the VEC coursera vourcher page and keep getting a "vec is down for maintenance" message. How long is the site going to be down for maintenance?"

There is no maintenance page up, so I'm not sure what they're seeing (could find out what URL they are on and what link they are clicking?).  We should follow up though. @koconnor719

**Comment by koconnor719** [comment]
These updated comments are not coming through GitHub which makes it impossible to manage updates.  i had thought all new comments would come thru GitHub. also they don't appear in user voice at the top of the list if i sort by date created?  they just get nested in old user voice messages.

**Comment by koconnor719** [comment]
coursers has replaced the blog w/ survey monkey feature and it now works correctly.
